sphinx_gallery/tests/test_gen_rst.py

```diff
@@ -448,6 +448,27 @@ def test_output_indentation(gallery_conf):
     assert output_test_string == test_string.replace(r"\n", "\n")
 
 
+def test_empty_output_box(gallery_conf):
+    """Tests that `print(__doc__)` doesn't produce an empty output box."""
+    compiler = codeop.Compile()
+    
+    code_block = ("code", "print(__doc__)", 1)
+
+    script_vars = {
+        "execute_script": True,
+        "image_path_iterator": ImagePathIterator("temp.png"),
+        "src_file": __file__,
+        "memory_delta": [],
+    }
+
+    example_globals = {'__doc__': ''}
+
+    output = sg.execute_code_block(
+        compiler, code_block, example_globals, script_vars, gallery_conf
+    )
+    assert output.isspace() == True
```

Comment:
Very minor comment: you don't need the `== True` here.

Reply:
Yes I realise this later - but excused myself as 'making it easier to read'? (:/) Too late now I guess?

Reply:
I pushed a commit directly in master so we can sleep better at night ;-).

Reply:
Thank you!